docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+<xode-block title="NodeJS">
+
+```ts 
+restartSubsystem(options?: RestartSubsystemOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```
+TODO 
+```
+</xode-block>
+</xode-group>
+
+## Enable a projection
+
+Enables an existing projection by name.
+You must have access to a projection to disable it.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Enable
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ enableProjection(projectionName: string, options?: EnableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Enabling a projection that does not exists results in an error: 
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#EnableNotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ enableProjection(projectionName: string, options?: EnableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Disable a projection 
+ 
+Disables a projection, this will save the projection checkpoint.
+You must have access to a projection to disable it.
+
+:::warning 
+The .net clients prior to version 21.6 had an incorrect behavior: they _will not_ save the checkpoint  
+:::
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Disable
+
+</xode-block>
+<xode-block title="Java">
+
+
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+disableProjection(projectionName: string, options?: DisableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Disabling a projection that does not exists results in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#DisableNotFound
+
+</xode-block>
+<xode-block title="Java">
+
+
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+disableProjection(projectionName: string, options?: DisableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+
+## Delete a projection
+
+Deletes a projection
```

Comment:
The same suggestion about giving context as in https://github.com/EventStore/documentation/pull/404/files#r689298658.